core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -378,7 +378,8 @@ private[spark] object JettyUtils extends Logging {
       server.getHandler().asInstanceOf[ContextHandlerCollection])
   }
 
-  private def createRedirectHttpsHandler(securePort: Int, scheme: String): ContextHandler = {
+  private def createRedirectHttpsHandler(
+      httpsConnector: ServerConnector, scheme: String): ContextHandler = {
```

Comment:
nit: one argument per line when using multiple lines.

But instead of changing this, why not pass the correct port from the caller in the first place?